geonotebook/config.py

```diff
@@ -39,10 +39,11 @@ def get_config(path=None):
 
 
 class Config(object):
-    _valid_vis_hash = {
-        "geoserver": vis.Geoserver,
-        "ktile": vis.Ktile
-    }
+    _valid_vis_hash = {}
+
+    @classmethod
+    def register(cls, name, server):
```

Comment:
I think this function could use a more specific name like `register_vis` in case we want to use a similar pattern for other extensions.